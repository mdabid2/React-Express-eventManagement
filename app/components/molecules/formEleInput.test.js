describe('Our first snapshot test', () => {
  it('Should compare the component with a snapshot', () => {
    const component = '<div>Hello Jest</div>'
    expect(component).toMatchSnapshot();
  })
});
