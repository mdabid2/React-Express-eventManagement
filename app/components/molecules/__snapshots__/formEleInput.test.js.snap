// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Our first snapshot test Should compare the component with a snapshot 1`] = `"<div>Hello Jest</div>"`;
